recipes/autoconf/all/test_v1_package/conanfile.py

```diff
@@ -1,15 +1,12 @@
 from conans import AutoToolsBuildEnvironment, ConanFile, tools
-from conan.tools.microsoft import is_msvc, unix_path
+from conan.tools.files import copy
+from conan.tools.microsoft import is_msvc
 import contextlib
 import os
-import shutil
-
-required_conan_version = ">=1.45.0"
 
 
 class TestPackageConan(ConanFile):
-    settings = "os", "compiler", "build_type", "arch"
-    exports_sources = "../test_package/configure.ac", "../test_package/config.h.in", "../test_package/Makefile.in", "../test_package/test_package_c.c", "../test_package/test_package_cpp.cpp",
+    settings = "os", "arch", "compiler", "build_type"
     test_type = "explicit"
 
```

Comment:
Shouldn't the `win_bash` be set as well for `test_v1_package` ?

Reply:
no, in conan v1 it's a different mechanism with win_bash in self.run directly.